modules/system.py

```diff
@@ -5,10 +5,7 @@
 def showMessage(message):
     window = TextWindow(bg=tidal.RED)
     window.cls()
-    x = 0
-    while x < len(message):
-        window.println(message[x:x+16])
-        x += 16
+    window.flow_lines(message)
```

Comment:
This doesn't actually print the text, it just splits it up into an array of strings the right length :)

I should probably add a print_lines API huh...